_includes/footer-phone.html

```diff
@@ -0,0 +1,12 @@
+{% if page.layout == 'content' or page.layout == 'content-statements'  %}
+{% assign small = 'small' %}
+{% endif %}
+
+<div class="footer-phone {{ small }} ">
```

Comment:
De headingstructuur is niet goed bruikbaar hier (zie [Headings are tables of contents](https://hiddedevries.nl/en/blog/2018-09-01-heading-structures-are-tables-of-contents)), wat er in de `h*` tags staat zijn geen headings.

Voorstel: 

* voeg `<h2 class="screen-reader-text">Contact</h2>` toe, zo kan iemand met screenreader makkelijker naar deze sectie springen
* gebruik voor de `footer-phone_text` en `footer-phone_action_text` een `<p>`, iets als: 

```
<p>{{ site.data.translations.footer-phone[page.lang] }}<br>
{{ site.data.translations.footer-phone-action[page.lang] }}</p>
```